<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryForm</class>
 <widget class="QWidget" name="QueryForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50, 50, 75);
color: white;
font-family: monospace;</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtrar por</string>
   </property>
  </widget>
  <widget class="QComboBox" name="filtro">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Emisor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Receptor</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Monto</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Fecha</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tipo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Igual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Minimo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Maximo</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Contiene</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inicia con</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Entre</string>
    </property>
   </item>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>260</y>
     <width>156</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="texto">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>100</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>y</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="montoCheckBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>131</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Totalizar monto</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
